lib/src/Base/Stat/TensorizedCovarianceModel.cxx

```diff
@@ -200,7 +200,8 @@ Matrix TensorizedCovarianceModel::partialGradient(const Point & s,
 /* Parameters accessor */
 void TensorizedCovarianceModel::setFullParameter(const Point & parameter)
 {
-  const UnsignedInteger parameterDimension = getParameter().getDimension();
+  const UnsignedInteger parameterDimension = getFullParameter().getSize();
+
```

Comment:
great! can you add mathias python testcase ?

Reply:
ok but a simplified one (without optim)